### PR TITLE
chore: `slog.slog` in `multichain-testing` ci  artifacts

### PR DESCRIPTION
**Pull request by 0xpatrickdev**
## Description
Include `slog.slog` in `multichain-testing` ci  artifacts

### Security Considerations
None

### Scaling Considerations
None

### Documentation Considerations
None

### Testing Considerations
Improves observability in CI.

### Upgrade Considerations
None

### CODE REVIEW

File: .github/workflows/multichain-e2e-template.yml [hunk 1]
```diff
@@ -102,6 +102,13 @@ jobs:
         env:
           FILE: ${{ inputs.config }}
 
+      - name: Capture slog.slog
+        if: always()
+        run: |
+          cd $GITHUB_WORKSPACE
+          echo "Capturing agoric slog.slog"
+          kubectl exec -i agoriclocal-genesis-0 -c validator -- cat slog.slog > "${LOG_DIR}/slog.slog" 2>&1 || true
```
Comment (0xpatrickdev) [inline .github/workflows/multichain-e2e-template.yml:110]: Seems like dealer's choice:

```sh
$ time kubectl exec -i agoriclocal-genesis-0 -c validator -- cat slog.slog > test.log
0.24s user 0.21s system 98% cpu 0.457 total

$ time kubectl cp ${NS}/agoriclocal-genesis-0:slog.slog test.log -c validator
tar: slog.slog: file changed as we read it
0.28s user 0.25s system 99% cpu 0.528 total
```

Comment (dckc) [reply .github/workflows/multichain-e2e-template.yml:110]: hm. yeah... `kubectl cp` seems like what we're doing... a comment about "file changed as we read it" might be nice.

Comment (dckc) [reply .github/workflows/multichain-e2e-template.yml:110]: I wonder how we do it for a3p.

**Review by dckc — APPROVED** [review]
looks good, presuming the file shows up in the ci artifacts

**Comment by 0xpatrickdev** [comment]
> looks good, presuming the file shows up in the ci artifacts

✅
image

**Pull request merged**